### ISSUE TITLE
No errors are reported in vue3

### ISSUE THREAD

**Issue by cuihaoweb** [TypeStrong/fork-ts-checker-webpack-plugin#783]
## Current behavior

No errors were detected in vue3

## Expected behavior
Report an error in the console

## Steps to reproduce the issue
This is used in vue3：
```js
<script setup lang="ts">
const num = ref<string>(0);
const addCount = () => {
    num.value++;
}
</script>
```
this is webpack.config.js：
```js
new ForkTsCheckerWebpackPlugin({
    typescript: {
        extensions: {
            vue: {
                disable: false,
                compiler: '@vue/compiler-sfc'
            }
        }
    }
}),
```
 num is a string, but ts does not report an error after ++

## Environment
 - "fork-ts-checker-webpack-plugin": ^7.2.13
 - "typescript": ^4.8.4
 - "ts-loader": ^9.4.1
 - "vue": "^3.2.41"

**Comment by timonbandit** [comment]
@cuihaoweb I'm sure you have an error in the config.
```javascript
new ForkTsCheckerWebpackPlugin({
    typescript: {
        extensions: {
            vue: {
                enabled: true,
                compiler: '@vue/compiler-sfc'
            }
        }
    }
}),
```
`disable: false` is wrong
`enabled: true` is correct

**Comment by tminich** [comment]
I have the same issue with Vue 2.7.14 and my config is (to my understanding) correct:
```json
{
    "async": false,
    "typescript": {
        "enabled": true,
        "extensions": {
            "vue": {
                "enabled": true,
                "compiler": "@vue/compiler-sfc"
            }
        },
        "memoryLimit": 4096,
        "profile": false,
        "diagnosticOptions": {
            "syntactic": true,
            "semantic": true,
            "declaration": false,
            "global": true
        }
    },
    "logger": {
        "infrastructure": "silent"
    }
}
```

**Comment by dojo-coder** [comment]
@timonbandit SFC script setup type checking does not work for Vue 3.
https://vuejs.org/api/sfc-script-setup.html
Works only with setup function. 
This ticket is a duplicate of:
https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/issues/772
This issue is currently blocking us from moving forward with esbuild as well.
